packages/ui/src/helpers/authenticator/facade.ts

```diff
@@ -171,6 +171,8 @@ export const getServiceContextFacade = (
         return 'verifyUser';
       case actorState?.matches('confirmVerifyUser'):
         return 'confirmVerifyUser';
+      case state.matches('signIn.runActor'):
+        return 'autoSignIn';
```

Comment:
Is this still necessary, since we're making minimal changes to the state machine now?

While I agree with minimizing error messages, I do not think `autoSignIn` is an accurate route for `runActor`, because `runActor` can happen in other transitions too. I think we'd prefer not to add this handling now and clean these off-states in later refactor altogether.

Reply:
OK I'll delete it. It still seems everything is working fine without this `autoSignIn` match

Reply:
I think it's still here 👀